Server/.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
So normally, if you've setup eslint, this ling is supposed to be flagged right. Since you're supposed to be setup for `ES6`.

Reply:
yes eslint is setup, but it's not flagging that line.
It was auto generated when I initialized the Sequelize